tests/baselines/reference/spreadInvalidArgumentType.errors.txt

```diff
@@ -47,33 +41,21 @@ tests/cases/compiler/spreadInvalidArgumentType.ts(55,17): error TS2698: Spread t
         var e: E;
     
         var o1 = { ...p1 };   // Error, generic type paramterre

```

Comment:
probably should just delete these since objectSpread covers the same cases

Reply:
Actually, the new cases don't cover `keyof T`, `T[K]`, mapped types over T, or intersections containing T. So those cases should move to objectSpread.